README.md

```diff
@@ -1,10 +1,15 @@
 [![Build Status](https://dev.azure.com/asottile/asottile/_apis/build/status/asottile.flake8-walrus?branchName=master)](https://dev.azure.com/asottile/asottile/_build/latest?definitionId=26&branchName=master)
 [![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/asottile/asottile/26/master.svg)](https://dev.azure.com/asottile/asottile/_build/latest?definitionId=26&branchName=master)
 
-flake8-walrus
-================
+# flake8-walrus
 
-flake8 plugin which forbids assignment expressions (the walrus operator)
+flake8 plugin which forbids assignment expressions (the walrus operator ``:=``).
```

Comment:
markdown is single backticks, also this shouldn't change as it matches the pypi description and repo description -- can you leave this alone?

Reply:
These minor differences between RST and mardown just bug me. Sigh.

Reply:
Heheh this one trips me up a lot as well, so much that I made a pre-commit hook for it (but in the other direction since I don't write rst all that often) : https://github.com/pre-commit/pygrep-hooks#provided-hooks (rst-backticks)